c/src/ml-api-inference-single.c

```diff
@@ -110,6 +110,7 @@ static const char *ml_nnfw_subplugin_name[] = {
   [ML_NNFW_TYPE_TVM] = "tvm",
   [ML_NNFW_TYPE_ONNX_RUNTIME] = "onnxruntime",
   [ML_NNFW_TYPE_NCNN] = "ncnn",
+  [ML_NNFW_TYPE_TENSORRT] = "tensorrt",
```

Comment:
I think it would be a good idea to use ```tensorRT``` instead of ```tensorrt```.  How do you feel about this?

Reply:
it is subplugin's name, lower case :)

Reply:
Oh~ , 😄 Thank you.